components/Landing/HeroMoment/LandingHeroMomentComponent.vue

```diff
@@ -2,37 +2,69 @@
   <article class="hero-moment">
     <LandingHeroMomentGrid>
       <div class="cds--grid hero-moment__container">
-        <LandingHeroMomentVersionInfo
-          class="hero-moment__version-info"
-          :version="version"
-        />
         <div class="cds--row">
           <h1
-            class="cds--col-md-3 cds--col-lg-5 cds--col-xlg-6 cds--col-max-7 hero-moment__title"
+            class="cds--col-md-4 cds--col-lg-5 cds--col-xlg-5 cds--col-max-7 hero-moment__title"
```

Comment:
I don't normally see the carbon grid classes being applied to a text element. Normally, they're classes of a surrounding `div`. 

I'd suggest

```html
<div class="cds--col-md-4 cds--col-lg-5 cds--col-max-7">
  <h1 class="hero-moment__title"></h1>
</div>
```